tests/aws/services/ec2/test_ec2.py

```diff
@@ -33,6 +39,24 @@ def create(template_name):
             aws_client.ec2.delete_launch_template(LaunchTemplateId=id)
 
 
+@pytest.fixture()
+def create_vpc(aws_client):
+    vpcs = []
+
+    def _create_vpc(
+        CidrBlock: str,
+        TagSpecifications: list[dict],
```

Comment:
Let's follow PEP8 and avoid camel case for argument names

```suggestion
        cidr_block: str,
        tag_specifications: list[dict],
```

Reply:
Done.